index.html

```diff
@@ -0,0 +1,2110 @@
+                    Actions related to the COVID-19 crisis, climate change and
+                    racial equality are to be among the first taken as he
+                    begins his tenure.
+                  </p>
+                </div>
+              </div>
+            </div>
+          </div>
+
+          <!-- END of Sponsored Insights section-->
+
+          <!-- END OF Featured Stories-->
+        </div>
+
+        <!-- END OF FIRST COLUMN -->
+
+        <!-- START OF SECOND COLUMN -->
+
+        <div class="top-story second-col col-12 col-md-7 col-lg-6 order-md-2">
+          <!-- START OF Top Story-->
+
+          <div class="block-title">Top story</div>
+          <div class="card border-0">
+            <div class="category">
+              <div class="card">
+                <div class="cate">
+                  <a href="/us">POLITICS</a>
+                </div>
+                <div class="image">
+                  <img class="card-img-top" src="asset/images/top-story.jpg" alt="Card image cap" />
+                </div>
+              </div>
+              <div class="card-body">
+                <h3 class="card-title">
+                  <a href="/trump-biden-super-predator-debate-1541565">
+                    Joe Biden's First Executive Actions Undo Many of Donald
+                    Trump's</a>
+                </h3>
+                <p class="card-text">
+                  Actions related to the COVID-19 crisis, climate change and
+                  racial equality are to be among the first taken as he begins
+                  his tenure.
+                </p>
+              </div>
+            </div>
+          </div>
+
+          <!-- END OF Top Story-->
+
+          <!-- START OF My turn and Cult&Trav-->
+
+          <div class="container-fluid">
+            <div class="row">
+              <div class="block-title">My turn</div>
+              <div class="col-12 col-md-4">
+                <div class="card border-0">
+                  <img class="card-img-top" src="asset/images/my-turn-1.jpg" alt="Card image cap" />
+                  <div class="card-body">
+                    <p class="card-text">
+                      I Married My Best Friend to Avoid Inheritance Tax
+                    </p>
+                  </div>
+                </div>
+              </div>
+
+              <div class="col-12 col-md-4">
+                <div class="card border-0">
+                  <img class="card-img-top" src="asset/images/my-turn-2.jpg" alt="Card image cap" />
+                  <div class="card-body">
+                    <p class="card-text">
+                      I've Rescued Hundreds of Special Needs Dogs
+                    </p>
+                  </div>
+                </div>
+              </div>
+              <div class="col-12 col-md-4">
+                <div class="card border-0">
+                  <img class="card-img-top" src="asset/images/my-turn-3.jpg" alt="Card image cap" />
```

Comment:
- [ ] The titles of the **My Turn** and **Culture and Title** articles should be links, the same as with the other article titles in the page.